File: beecrowd_exercicios_em_Python/ad-hoc/gangorra.py
def gangorra(a, b, c, d):
    if a * b == c * d:
        return 0
    if a * b > c * d:
        return -1

    return 1


a, b, c, d = map(int, input().split())

print(gangorra(a, b, c, d))

File: beecrowd_exercicios_em_Ruby/ad_hoc/gangorra.rb
def gangorra(a,b,c,d)
  if a*b == c*d
    return 0
  end

  if a*b > c*d
    return -1
  end

  1
end

a,b,c,d = gets.split(' ').map(&:to_i)

puts gangorra(a,b,c,d)
